src/publishers/github.js

```diff
@@ -31,7 +31,8 @@ export default async (artifacts, packageJSON, forgeConfig, authToken, tag) => {
           repo: forgeConfig.github_repository.name,
           tag_name: tag || `v${packageJSON.version}`,
           name: tag || `v${packageJSON.version}`,
-          draft: true,
+          draft: forgeConfig.github_repository.draft !== false,
```

Comment:
Sure, but the github repository name and owner can't be declared at the CLI right now either.